manim/utils/cfg_file_utils.py

```diff
@@ -1,28 +1,30 @@
 """
-cfgwriter.py
+cfg_file_utils.py
 ------------
 
-Inputs the configuration files while checking it is valid. Can be executed by `manim-cfg` command.
+General Config File Managing Utilities.
+The functions below can be called via the `manim cfg` subcommand.
 
 """
 import os
 import configparser
 
-from .config_utils import _run_config, _paths_config_file
+from .config_utils import _run_config, _paths_config_file, curr_config_dict
 
 from rich.console import Console
 from rich.progress import track
 from rich.style import Style
 from rich.errors import StyleSyntaxError
 
-__all__ = ["main"]
+__all__ = ["write","show","export"]
 
 INVALID_STYLE_MSG = "[red bold]Your Style is not valid. Try again.[/red bold]"
 INTRO_INSTRUCTIONS = """[red]The default colour is used by the input statement.
 If left empty, the default colour will be used.[/red]
-[magenta] For a full list of styles, visit[/magenta] https://rich.readthedocs.io/en/latest/style.html"""
+[magenta] For a full list of styles, visit[/magenta] [green]https://rich.readthedocs.io/en/latest/style.html[/green]"""
```

Comment:
Suggestion: an alternative is to change the background color instead of the face color. Not sure if that'd look good tho.

Reply:
I think it might be a bit jarring, but that's just me.